shoop/campaigns/models/product_effects.py

```diff
@@ -17,7 +17,7 @@ class ProductDiscountEffect(PolymorphicShoopModel):
     model = None
     admin_form_class = None
 
-    campaign = models.ForeignKey("CatalogCampaign", related_name='effects', verbose_name=_("campaign"))
+    campaign = models.ForeignKey("CatalogCampaign", related_name='product_effects', verbose_name=_("campaign"))
```

Comment:
IMHO `effects` was better related name. Of course they are effects of a product, since it's a catalog campaign (Catalog is about products ;) )